content/posts/meetings.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Meetings
+permalink: /meetings/
+header_image: meetings
+---
+
+# Community Meetings
+NUTS maakt het mogelijk dat softwareapplicaties in de zorg met elkaar op een vertrouwelijke, veilige en toegankelijke manier kunnen communiceren. Tijdens onze maandelijkse communitymeetings praten wij je bij over Nuts en aan Nuts gerelateerde onderwerpen.   
```

Comment:
Misschien niet nodig om hier in het kort te vertellen wat Nuts mogelijk maakt, maar misschien wel handig om iets meer over de community meetings te vertellen. Ook waar de agenda staat en hoe je zelf iets kunt vertellen?